lib/worker.js

```diff
@@ -10,11 +10,11 @@ const { once } = require('events')
 
 /* istanbul ignore file */
 
-module.exports = async function (transports) {
-  transports = await Promise.all(transports.map(async (t) => {
+module.exports = async function ({ destinations }) {
+  destinations = await Promise.all(destinations.map(async (t) => {
```

Comment:
`Promise.all` could throw and I'm not seeing where it is handled. Should this be `Promise.allSettled`?

Reply:
It's handled by https://github.com/pinojs/thread-stream/blob/62d24c681c673aad919bd79fed24e6b75bb92d28/lib/worker.js#L91.

I would still need to throw... in that case what error should I throw if I have more than one?

Reply:
So that I'm clear: that `process.on('unhandledRejection')` covers _only_ the worker child process?

Reply:
yes

Reply:
Then I am good with the code as-is.